src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <img class="logomoyo"
+           src="https://github.com/hserpukhovitina/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+           alt="Moyo"
+      >
+      <nav class="nav">
+        <a class="nav-link is-active" href="#">apple</a>
+        <a class="nav-link is-active" href="#">samsung</a>
+        <a class="nav-link is-active" href="#">смартфоны и телефоны</a>
```

Comment:
```suggestion
        <a class="nav-link is-active" href="#">apple</a>
        <a class="nav-link " href="#">samsung</a>
        <a class="nav-link" href="#">смартфоны и телефоны</a>
```
Клас `is-active` має бути тільки для першого пункту